contracts/programs/ocr2/src/lib.rs

```diff
@@ -159,11 +168,15 @@ pub mod ocr2 {
             ctx.accounts.token_vault.to_account_info(),
             ctx.accounts.vault_authority.clone(),
```

Comment:
Why `clone()` vs `to_account_info()`?

Asking because I've noticed we use `ctx.accounts.vault_authority.to_account_info()` [here#L73](https://github.com/smartcontractkit/chainlink-solana/blob/7108b0b3954e0be023bac74bceb3a7b749f90b8a/contracts/programs/ocr2/src/lib.rs#L73)

Reply:
They're basically the same:

```
    fn to_account_info(&self) -> AccountInfo<'info> {
        self.as_ref().clone()
    }

```

I'll switch to `to_account_info` for consistency